lib/broadside/configuration/deploy_config.rb

```diff
@@ -22,14 +22,18 @@ class DeployConfig < ConfigStruct
         :env_vars,
         :command,
         :instance,
-        :predeploy_commands
+        :predeploy_commands,
+        :service_config,
+        :task_definition_config
       )
 
       TARGET_ATTRIBUTE_VALIDATIONS = {
         scale: ->(target_attribute) { validate_types([Fixnum], target_attribute) },
         env_file: ->(target_attribute) { validate_types([String], target_attribute) },
         command: ->(target_attribute) { validate_types([Array, NilClass], target_attribute) },
-        predeploy_commands: ->(target_attribute) { validate_predeploy(target_attribute) }
+        predeploy_commands: ->(target_attribute) { validate_predeploy(target_attribute) },
+        service_config: ->(target_attribute) { validate_types([Hash, NilClass], target_attribute) },
```

Comment:
originally intended to support things outside of ECS, but whatever. let's just go with this and ditch that config file. This will have to be a major version bump since that will be a breaking change.